client/src/app.tsx

```diff
@@ -40,13 +71,28 @@ const App = ({rep, undoManager}: AppProps) => {
   const [detailIssueID, setDetailIssueID] = useIssueDetailState();
   const [menuVisible, setMenuVisible] = useState(false);
 
-  const [state, dispatch] = useReducer(timedReducer, {
-    allIssuesMap: new Map(),
-    viewIssueCount: 0,
-    filteredIssues: [],
-    filters: getFilters(view, priorityFilter, statusFilter),
-    issueOrder: getIssueOrder(view, orderBy),
-  });
+  const [filters, setFilters] = useState(
+    getFilters(view, priorityFilter, statusFilter),
+  );
+  const [issueOrder, setIssueOrder] = useState(getIssueOrder(view, orderBy));
+
+  const [, filterdIssues] = useQuery(() => {
```

Comment:
`useQuery`, since it uses an effect to compute the view, causes a double render problem.

note to self: update `useQuery` to emulate an effect via `useState`

e.g.,

```ts
function useQuery(q, deps) {
  const [oldDeps, setOldDeps] = useState();
  const [view, setView] = useSate();
  if (!shallowEqual(deps, oldDeps) {
    destroy(view);
    setView(q());
  }

  useEffect(() => {
    destroy(view);
  }, []);

  return view;
}
```

Reply:
Yep, this worked. All flashing is gone.